docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -2,4 +2,173 @@
+anti-patterns in the code; ``make typecheck`` offers a more nuanced look
+at interfaces between functions, methods, etc. and may spot issues that
+would cause ``make unittest`` to fail.
+
+The ``make test`` command is also automatically executed when ``conda``
+builds a ``uwtools`` package, so it is important to periodically run
+these tests during development and, crucially, before merging changes,
+to ensure that the tests will pass when CI builds the ``workflow-tools``
+code.
+
+In addition to the ``make devshell`` command, two other ``make`` targets
+are available for use *outside* a development shell, i.e. from the base
+conda environment (requires presence of the ``condev`` package):
+
+
++------------------+-------------------------------------------------------+
+| Command          | Description                                           |
++==================+=======================================================+
+| ``make env``     | Creates a conda environment based on the ``uwtools``  |
+|                  | code                                                  |
++------------------+-------------------------------------------------------+
+| ``make meta``    | Update ``recipe/meta.json`` from ``recipe/meta.yaml`` |
++------------------+-------------------------------------------------------+
+| ``make package`` | Builds a ``uwtools`` conda package                    |
++------------------+-------------------------------------------------------+
+
+
+These targets work from the code in its current state in the clone.
+``make env`` calls ``make package`` automatically to create a local
+package, then builds an environment based on the package.
+
+Building condev locally
+-----------------------
+
+As an alternative to installing a prebuilt package from
+`anaconda.org`_, the ``condev`` package can be
+built locally, then installed into the local conda installation. Ensure
+that ``conda-build`` and ``conda-verify`` are installed in the base
+environment.
+
+.. code::
+
+   # Activate your conda
+   git clone https://github.com/maddenp/condev.git
+   make -C condev package
+   conda install -y -c local condev
+
+Files derived from condev
+-------------------------
+
+The following files in this repo are derived from their counterparts in
+the `condev demo`_:
+
+
+.. code::
+
+   ├── Makefile
+   ├── recipe
+   │   ├── build.sh
+   │   ├── channels
+   │   ├── meta.json
+   │   ├── meta.yaml
+   │   └── run_test.sh
+   ├── src
+   │   ├── pyproject.toml
+   │   ├── setup.py
+
+Behaviors described in previous sections may rely on these files
+continuing to follow ``condev`` conventions.
+
+
+.. _External Links:
+
+.. _anaconda.org: https://anaconda.org
+.. _condev: https://github.com/maddenp/condev
+.. _condev demo: https://github.com/maddenp/condev/tree/main/demo
+.. _Miniforge3: https://github.com/conda-forge/miniforge#download
+.. _workflow-tools repository: https://github.com/ufs-community/workflow-tools
```

Comment:
I chose not to do inline external links, but it is a possibility if we like it better. It only affects the rst documentation and not what will appear to the user.

It's the difference between:

**Version #1:**
- "Download, install, and activate the latest `'Miniforge3'_` for your system."
- .. _Miniforge3: https://github.com/conda-forge/miniforge#download

**Version #2:**
- Download, install, and activate the latest `'Miniforge3 <https://github.com/conda-forge/miniforge#download>'__`
   for your system.

Reply:
I like the choice you made. Do you know if it's possible to define these links in a common file that could be shared between `.rst` files? I ask because, for example, I think we'll want to talk about -- and link to -- tools like `pylint`, `black`, etc. in multiple places, and having one canonical set of links that we could refer to, so that all our links are consistent, would be great.